### ISSUE TITLE
Test data 

### ISSUE THREAD

**Issue by abhishekju06** [WenjieDu/SAITS#22]
Hi,

After certain modification and inclusion of code snippets I was able train, validate and get the mae for test data.
I want to obtain the de-normalized value after the imputation happens in test data, both predicted and actual. Can you help?

**Comment by WenjieDu** [comment]
Hi there,<br><br>Thank you so much for your attention to SAITS! If you find SAITS is helpful to your work, please star⭐️ this repository. Your star is your recognition, which can let others notice SAITS. It matters and is definitely a kind of contribution.<br><br>I have received your message and will respond ASAP. Thank you again for your patience! 😃<br><br>Best,<br>Wenjie

**Comment by WenjieDu** [comment]
In data preprocessing, you normalized data with a scaler from sklearn. Keep it. After imputation, you inverse the normalization with func `inverse_transform()`. For example, [StandardScaler.inverse_transform()](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html#sklearn.preprocessing.StandardScaler.inverse_transform).

**Comment by abhishekju06** [comment]
Hi,
I was able to generate Actual Vs Predicted in csv format. The results are pretty amazing! Thanks for ur patience wid me.

**Comment by WenjieDu** [comment]
@abhishekju06 Glad to hear that! I sincerely invite you to follow me on GitHub so that you can receive PyPOTS' latest news instantly!
